disnake/reaction.py

```diff
@@ -200,4 +204,124 @@ def users(
         if limit is None:
             limit = self.count
 
-        return ReactionIterator(self.message, emoji, limit, after)
+        return ReactionIterator(self.message, emoji, limit, after, type=0)
+
+
+class BurstReaction(Reaction):
+    """Represents a Super reaction to a message.
+
+    .. versionadded:: 2.9
+
+    This class is a subclass of :class:`.Reaction`.
+
+    Depending on the way this object was created, some of the attributes can
+    have a value of ``None``.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two Super reactions are equal. This works by checking if the
+            emoji is the same. So two messages with the same reaction will be
+            considered "equal".
+
+        .. describe:: x != y
+
+            Checks if two Super reactions are not equal.
+
+        .. describe:: hash(x)
+
+            Returns the Super reaction's hash.
+
+        .. describe:: str(x)
+
+            Returns the string form of the Super reaction's emoji.
+
+    Attributes
+    ----------
+    emoji: Union[:class:`Emoji`, :class:`PartialEmoji`, :class:`str`]
+        The Super reaction emoji. May be a custom emoji, or a unicode emoji.
+    count: :class:`int`
+        Number of times this Super reaction was made.
+    me: :class:`bool`
+        If the user sent this Super reaction.
+    message: :class:`Message`
+        The message this Super reaction belongs to.
+    burst_colors: List[:class:`Colour`]
+        The list of :class:`Colour` used for super reaction.
+
+        .. note::
+
+            This information is only available via :meth:`abc.Messageable.fetch_message`.
+    """
+
+    __slots__ = "burst_colors"
```

Comment:
```suggestion
    __slots__ = ("burst_colors",)
```
small nit - both technically work, the convention in this repo is to always use a tuple